crates/oxc_ast/src/ast/js.rs

```diff
@@ -1709,6 +1709,7 @@ pub struct FunctionBody<'a> {
 )]
 #[derive(Debug)]
 #[generate_derive(CloneIn, GetSpan, GetSpanMut, ContentEq, ESTree)]
+#[estree(add_fields(generator = false, id = null))]
```

Comment:
It might be okay to write it directly as rust expression? Also `add_ts` is required for .d.ts output.

```suggestion
#[estree(add_fields(generator = false, id = Option::<()>::None), add_ts="generator: false; id: null;")]
```

Reply:
Thanks for getting involved @camchenry.

`()` serializes to `null`, so that's probably the simplest way:

```rs
#[estree(add_fields(generator = false, id = (), add_ts="generator: false; id: null;")]
```

Reply:
Oh perfect, I think adding an alias here makes a lot of sense. It's intuitive to me that `None` would serialize to that, but it's not easy to use everywhere, but `()` is better.